<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1704</width>
    <height>1026</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Non-Linear Chromaticity GUI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>1501</width>
      <height>981</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="timberTab">
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <attribute name="title">
      <string>Timber</string>
     </attribute>
     <widget class="QPushButton" name="extractTimberButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>241</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Extract Data</string>
      </property>
     </widget>
     <widget class="QLabel" name="statusTimberExtractionLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>241</width>
        <height>141</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter|Qt::AlignTop</set>
      </property>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>301</width>
        <height>89</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="2" column="1">
        <widget class="QDateTimeEdit" name="endTimeTimberEdit">
         <property name="date">
          <date>
           <year>2022</year>
           <month>1</month>
           <day>1</day>
          </date>
         </property>
         <property name="displayFormat">
          <string>yyyy-MM-dd HH:mm:ss</string>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="toolTip">
          <string>Start time of measurement, in local time</string>
         </property>
         <property name="text">
          <string>Start of Measurement</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDateTimeEdit" name="startTimeTimberEdit">
         <property name="dateTime">
          <datetime>
           <hour>0</hour>
           <minute>0</minute>
           <second>0</second>
           <year>2022</year>
           <month>1</month>
           <day>1</day>
          </datetime>
         </property>
         <property name="displayFormat">
          <string>yyyy-MM-dd HH:mm:ss</string>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="toolTip">
          <string>End time of measurement, in local time</string>
         </property>
         <property name="text">
          <string>End of Measurement</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Extract raw BBQ</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QCheckBox" name="rawBBQCheckBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="tristate">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="cleaningTab">
     <attribute name="title">
      <string>Cleaning</string>
     </attribute>
     <widget class="QPushButton" name="createPlateausButton">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>110</y>
        <width>121</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Create Plateaus</string>
      </property>
     </widget>
     <widget class="QTabWidget" name="cleaningPlotsTab">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>1481</width>
        <height>791</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="beam1PlotsTab">
       <attribute name="title">
        <string>Beam 1</string>
       </attribute>
       <widget class="QTabWidget" name="tabCleaningB1">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1471</width>
          <height>751</height>
         </rect>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="tabBarAutoHide">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>DPP</string>
         </attribute>
         <widget class="QWidget" name="horizontalLayoutWidget">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>1461</width>
            <height>711</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="cleaningPlotB1Layout"/>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_5">
         <attribute name="title">
          <string>Raw Data</string>
         </attribute>
         <widget class="QWidget" name="horizontalLayoutWidget_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>1461</width>
            <height>711</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="rawPlotB1Layout"/>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Cleaned Data</string>
         </attribute>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="beam2PlotsTab">
       <attribute name="title">
        <string>Beam 2</string>
       </attribute>
       <widget class="QTabWidget" name="tabCleaningB2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1471</width>
          <height>751</height>
         </rect>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="tabBarAutoHide">
         <bool>false</bool>
        </property>
        <widget class="QWidget" name="tab_3">
         <attribute name="title">
          <string>DPP</string>
         </attribute>
         <widget class="QWidget" name="horizontalLayoutWidget_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>1461</width>
            <height>711</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="cleaningPlotB2Layout"/>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_6">
         <attribute name="title">
          <string>Raw Data</string>
         </attribute>
         <widget class="QWidget" name="horizontalLayoutWidget_4">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>1461</width>
            <height>711</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="rawPlotB2Layout"/>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_4">
         <attribute name="title">
          <string>Cleaned Data</string>
         </attribute>
        </widget>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="cleaningWidget" native="true">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>10</y>
        <width>401</width>
        <height>151</height>
       </rect>
      </property>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>60</y>
         <width>80</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>281</width>
        <height>91</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="toolTip">
          <string>The RF Frequency isn't always registered for both beams. Usually only BEAM 1 contains the data.</string>
         </property>
         <property name="text">
          <string>Beam used for RF</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="rfBeamComboBox">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="insertPolicy">
          <enum>QComboBox::InsertAtBottom</enum>
         </property>
         <property name="modelColumn">
          <number>0</number>
         </property>
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Refine Start</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDateTimeEdit" name="startPlateauDateTimeEdit">
         <property name="displayFormat">
          <string>yyyy-MM-dd HH:mm:ss</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Refine End</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDateTimeEdit" name="endPlateauDateTimeEdit">
         <property name="displayFormat">
          <string>yyyy-MM-dd HH:mm:ss</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="chromaticityTab">
     <attribute name="title">
      <string>Chromaticity</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="correctionTab">
     <attribute name="title">
      <string>Correction</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>181</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="measurementLabel">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Measurement Settings</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="1">
        <widget class="QLineEdit" name="nominalRfLineEdit">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="nominalRfLabel">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Nominal Frequency of the RF (Usually around 400_788_869).&lt;/p&gt;&lt;p&gt;If set to 0 or &lt;span style=&quot; font-style:italic;&quot;&gt;None&lt;/span&gt;, it will default to the first value of the measurement.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>Nominal RF</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="saveSettingsButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Settings</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>181</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="modelLabel">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Model information. Click for more details.</string>
       </property>
       <property name="text">
        <string>Model</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QLineEdit" name="alfaB1LineEdit">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="alfaB1Label">
         <property name="text">
          <string>ALFA B1</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="alfaB2Label">
         <property name="text">
          <string>ALFA B2</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="alfaB2LineEdit">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1704</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionNew</sender>
   <signal>triggered()</signal>
   <receiver>mainWindow</receiver>
   <slot>newMeasurementClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>849</x>
     <y>512</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpen</sender>
   <signal>triggered()</signal>
   <receiver>mainWindow</receiver>
   <slot>openMeasurementClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>849</x>
     <y>512</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>mainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>849</x>
     <y>512</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>extractTimberButton</sender>
   <signal>clicked()</signal>
   <receiver>mainWindow</receiver>
   <slot>extractTimberClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>342</x>
     <y>173</y>
    </hint>
    <hint type="destinationlabel">
     <x>851</x>
     <y>512</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>createPlateausButton</sender>
   <signal>clicked()</signal>
   <receiver>mainWindow</receiver>
   <slot>createPlateausClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>322</x>
     <y>173</y>
    </hint>
    <hint type="destinationlabel">
     <x>851</x>
     <y>512</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveSettingsButton</sender>
   <signal>clicked()</signal>
   <receiver>mainWindow</receiver>
   <slot>saveSettingsClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>95</x>
     <y>284</y>
    </hint>
    <hint type="destinationlabel">
     <x>851</x>
     <y>512</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>newMeasurementClicked()</slot>
  <slot>openMeasurementClicked()</slot>
  <slot>closeGui()</slot>
  <slot>extractTimberClicked()</slot>
  <slot>createPlateausClicked()</slot>
  <slot>saveSettingsClicked()</slot>
 </slots>
</ui>
